projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/close-account.core-e2e-spec.ts

```diff
@@ -69,6 +69,8 @@ describe('My Account - Close Account', () => {
         alerts
           .getSuccessAlert()
           .should('contain', 'Account closed with success');
+
```

Comment:
@Zeyber the e2e test does cover verify that the page is redirected to the homepage above on line 67

Reply:
This line `cy.location('pathname').should('contain', '/');` will pass with any url that contains a `/` character which doesn't really mean anything. Instead we should look for elements unique to the home page.